### DISCUSSION TITLE
Date race risk causing at new xcode.

### DISCUSSION THREAD

**Discussion by rezojoglidze** [pointfreeco/swift-composable-architecture#3384] [General]
### Description

Date race risk causing at new xcode.

<img width="303" alt="image" src="https://github.com/user-attachments/assets/c3229645-6d26-4713-bf08-a2a56c594847">

### Checklist

- [ ] I have determined whether this bug is also reproducible in a vanilla SwiftUI project.
- [ ] If possible, I've reproduced the issue using the `main` branch of this package.
- [X] This issue hasn't been addressed in an [existing GitHub issue](https://github.com/pointfreeco/swift-composable-architecture/issues) or [discussion](https://github.com/pointfreeco/swift-composable-architecture/discussions).

### The Composable Architecture version information

1.15.0

### Destination operating system

iOS 18

### Xcode version information

Version 16.1 beta (16B5001e)

**Comment by mbrandonw** [comment]
Hi @rezojoglidze, Xcode 16.1 beta uses an older version of Swift that has a bug in sendability checking (it's the same version of Swift shipped with Xcode 16.0 beta 5). If you use the Xcode 16.0 RC you will not have this problem.

Since this isn't an issue with the library I am going to convert it to a discussion. Please feel free to continue the conversation over there!

**Reply by rezojoglidze** [reply]
Everything is clear. Thank you.
